<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QDialog" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>首选项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_openReverse">
     <property name="text">
      <string>打开顺序反转</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_finishRAWPhicture">
     <property name="text">
      <string>拼接后自动删除原图片</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>保存格式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_saveType">
       <item>
        <property name="text">
         <string>PNG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JPG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>拼接方式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_splicingType">
         <item>
          <property name="text">
           <string>低分辨率放大</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高分辨率缩小</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>直接拼接</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_paddingType">
         <property name="text">
          <string>填充颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_paddingType">
         <item>
          <property name="text">
           <string>透明</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>黑色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>白色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深灰色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>灰色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>浅灰色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>红色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>绿色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>蓝色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>青色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>洋红色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>黄色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深红色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深绿色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深蓝色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深青色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深洋红色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深黄色</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>放大方式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_narrowType">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>双线性插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>最近邻插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>使用像素区域关系进行重采样</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4x4像素邻域的双三次插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8x8像素邻域的Lanczos插值</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>缩小方式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_shrinkType">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>双线性插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>最近邻插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>使用像素区域关系进行重采样</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4x4像素邻域的双三次插值</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8x8像素邻域的Lanczos插值</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>自动拼接特征点检测方式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_featureExtraction">
       <item>
        <property name="text">
         <string>SIFT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SURF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ORB</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
